velox/functions/sparksql/MapFromArrays.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/functions/sparksql/MapFromArrays.h"
+
+namespace facebook::velox::functions::sparksql {
+
+std::shared_ptr<exec::VectorFunction> makeMapFromArrays(
```

Comment:
Can we add function in Map.h makeMapFromArrays with argument (bool AllowDuplicateKeys, DeduplicateKeys), and function getMapFromArraysSignature then we can move class makeMapFromArrays to cpp file. What do you think about that? @rui-mo

Reply:
I'm not sure I got the point. Do we need the `getMapFromArraysSignature` function, can it just be wrapped in the call of `registerStatefulVectorFunction`?